server/dataGen.js

```diff
@@ -108,9 +108,9 @@ const dataGen = () => {
     };
     hundredRecords.push(home);
   }
-  Home.sync({ force: true }).then(() => {
+  return Home.sync({ force: true }).then(() => {
     // Table created
-    Home.bulkCreate(hundredRecords);
+    return Home.bulkCreate(hundredRecords);
```

Comment:
Since you're returning this promise, you can omit the curly braces and return word (this is part of the AirBnb style guide, see item 8.2)

Reply:
Changed to a single line without return statement.